tcms/testcases/templates/testcases/mutable.html

```diff
@@ -0,0 +1,152 @@
+                            </option>
+                        {% endfor %}
+                    </select>
+                    {{ form.product.errors }}
+                </div>
+
+                <div class="col-md-1 col-lg-1">
+                    <label for="id_category">{% trans "Category" %}</label>
+                    <a href="{% url 'admin:testcases_category_add' %}?_popup" id="add_id_category" alt="{% trans 'add new Category' %}" title="{% trans 'add new Category' %}">+</a>
+                </div>
+                <div class="col-md-3 col-lg-3 {% if form.category.errors %}has-error{% endif %}">
+                    <select id="id_category" name="category" class="form-control selectpicker">
+                        {% for category in form.category.field.queryset %}
+                            <option value="{{ category.pk }}" {% if category.pk == form.category.value %}selected{% endif %}>
+                                {{ category.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                    {{ form.category.errors }}
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_status">{% trans "Status" %}</label>
+                <div class="col-md-3 col-lg-3 {% if form.case_status.errors %}has-error{% endif %}">
+                    <select id="id_status" name="status" class="form-control selectpicker">
+                        {% for case_status in form.case_status.field.queryset %}
+                            <option value="{{ case_status.pk }}" {% if case_status.pk == form.case_status.value %}selected{% endif %}>
+                                {{ case_status.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                    {{ form.case_status.errors }}
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_priority">{% trans "Priority" %}</label>
+                <div class="col-md-3 col-lg-3 {% if form.case_status.errors %}has-error{% endif %}">
+                    <select id="id_priority" name="priority" class="form-control selectpicker">
+                        {% for priority in form.priority.field.queryset %}
+                            <option value="{{ priority.pk }}" {% if priority.pk == form.priority.pk %}selected{% endif %}>
+                                {{ priority.value }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                    {{ form.priority.errors }}
+                </div>
+
+                <label class="col-md-1 col-lg-1">{% trans "Automated" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <input class="bootstrap-switch" name="is_automated" type="checkbox">
+                </div>
+            </div>
+
```

Comment:
Design feedback:
- Script/Arguments are for automated tests
- Reference llink/Requirements will usually be links to external documents, supposedly feature requirements documents.

so grouping should be 2x2 in this case.

- Notes is a text area field in its own right so it must spam the entire page, but not be very high (not sure what the default is).

All of these 5 fields should be moved under the Text field b/c they are less frequently used.

Reply:
I've merged the changes for is_automated so it should work.

Reply:
When creating new TC from a TP you need a help link under summary, pointing back to the test plan. See testruns/mutable.html for example.

Changes for `is_automated` have been merged!